units/stats_latency.go

```diff
@@ -0,0 +1,60 @@
+package units
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/amboy/job"
+	"github.com/mongodb/amboy/registry"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/logging"
+	"github.com/pkg/errors"
+)
+
+const (
+	latencyStatsCollectorJobName  = "latency-stats-collector"
+	latencyStatsCollectorInterval = time.Minute
+)
+
+func init() {
+	registry.AddJobType(latencyStatsCollectorJobName,
+		func() amboy.Job { return makeLatencyStatsCollector() })
+}
+
+type latencyStatsCollector struct {
+	job.Base `bson:"job_base" json:"job_base" yaml:"job_base"`
+	logger   grip.Journaler
```

Comment:
you only need to have a logger thing here if you want to write a test that mocks out the logger and inject that after creation, otherwise you can just use the global instance.

Reply:
done